app.js

```diff
@@ -4,6 +4,7 @@ import { didUserWin } from './check-results.js';
 
 // grabbing HTML elements
 const computerDisplay = document.getElementById('computer-display');
+const computerImage = document.getElementById('computer-image');
```

Comment:
Great attention to detail. As a user, I definitely want to know what the computer got.